### PR TITLE
🔖(chore) bump release to version 2.10.0

### PR DESCRIPTION
**Pull request by sampaccoud**
## Added

- Add infinite scrolling to the facet search modal in course search.
- Add reusable form components
- Fix blogpost header image aspect ratio and sizes.
- Add theme property to `Spinner` component
- Add setting to limit the number of archived course runs displayed by default on a course detail page.
- Add timing function variables
- Add a `outline` variant to button
- Create a `shadowed-box` mixin
- Add a meta description default value from course introduction, blog excerpt, category description, person bio, program excerpt and organization description
- Improve social sharing by specifying og:description meta for blogpost, category, person, program and organization.

## Changed

- Include new pages in navigation by default so they have a breadcrumb
- Create a generic React Modal component
- Create an context util
- Add react-query to manage API requests and local data store
- Move type utils into type directory
- Make licenses on course page optional

## Fixed

- Fix negative scores causing a 500 error for courses far in the future.
- Remove transparency when not necessary so that images are converted to jpg
- Prevent logout action to be called twice
- Fix og:image when using S3Boto3Storage
- Remove error message related with cache when any test fails

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+## [2.10.0] - 2021-12-17
```
Comment (lunika) [inline CHANGELOG.md:11]:
```suggestion
## [2.10.0] - 2021-12-27
```

**Review by lunika — APPROVED** [review]

**Comment by lunika** [comment]
You don't add in the changelog the new russian available translations ?

File: CHANGELOG.md (hunk 1 above)
Comment (sampaccoud) [reply CHANGELOG.md:11]: lol :ok_hand:

**Comment by sampaccoud** [comment]
> You don't add in the changelog the new Russian available translations ?

Good idea!

**Pull request merged**